api/apierr/service.go

```diff
@@ -0,0 +1,27 @@
+package apierr
+
+import (
+	"errors"
+	"net/http"
+
+	"github.com/shellhub-io/shellhub/api/apicontext"
+)
+
+var ErrUnauthorized = errors.New("unauthorized")
+var ErrUserNotFound = errors.New("user not found")
+var ErrResourceNotFound = errors.New("resource not found")
+var ErrDuplicateID = errors.New("user already member of this namespace")
+var ErrUserOwner = errors.New("cannot remove this user")
+
+func HandleError(c apicontext.Context, err error) error {
```

Comment:
`ToApiError`

Reply:
HandleError handles errors and writes them according to our API conventions.